datacenterlight/templates/datacenterlight/base.html

```diff
@@ -1,4 +1,4 @@
-{% load staticfiles i18n%}
+{% load staticfiles i18n cms_tags sekizai_tags %}
```

Comment:
Just wondering: are we using base.html in non-cms pages and if yes, how does django our rendering behave in that case with the cms_tags present?